File: Zadanie1.py
a = float(input("Podaj pierwszą liczbę: "))
b = float(input("Podaj drugą liczbę: "))
operation = input("Wybierz działanie (+, -, *, /): ")

if operation == "+":
    result = a+b
elif operation == "-":
    result = a-b
elif operation == "*":
    result = a*b
elif operation == "/":
    if b != 0:
        result = a/b
    else:
        print("Nie można dzielić przez zero")
        exit()
else:
    print("Błędny znak")
    exit()

print("Wynik: ", result)

File: Zadanie1.cs
using System;

class Program
{
    static void Main()
    {
        Console.Write("Podaj pierwszą liczbę: ");
        double a = Convert.ToDouble(Console.ReadLine());

        Console.Write("Podaj drugą liczbę: ");
        double b = Convert.ToDouble(Console.ReadLine());

        Console.Write("Wybierz operację (+, -, *, /): ");
        string operation = Console.ReadLine();

        double result;

        if (operation == "+")
            result = a+b;
        else if (operation == "-")
            result = a-b;
        else if (operation == "*")
            result = a*b;
        else if (operation == "/")
        {
            if (b != 0)
                result = a/b;
            else
            {
                Console.Write("Nie można dzielić przez zero");
                return;
            }
        }
        else
        {
            Console.Write("Błędny znak");
            return;
        }

        Console.Write("Wynik: " + result);
    }
}